react-app-rewired/headers/csps/jupiter.ts

```diff
@@ -0,0 +1,5 @@
+import type { Csp } from '../types'
+
+export const csp: Csp = {
+  'connect-src': ['https://quote-api.jup.ag/'],
```

Comment:
It makes total sense, CSP are not updated if we update the env variable

I would like to avoid fixing all the other cases, can we create an issue to fix all the other cases?

Jup handled in [e4891d6](https://github.com/shapeshift/web/pull/8120/commits/e4891d6256da07fae14a9f7eaa0e89ba7aefbe0d)

Reply:
Agree, lets not let this PR turn into "fix: shapeshift" 😅